QUICKSTART.md

```diff
@@ -0,0 +1,354 @@
+kubectl --context kind-florin kubectl get pods -A
+```
+
+which should yield something like:
+
+```console
+NAMESPACE                         NAME                                               READY   STATUS    RESTARTS   AGE
+kcp-edge-syncer-florin-1yi5q9c4   kcp-edge-syncer-florin-1yi5q9c4-77cb588c89-xc5qr   1/1     Running   0          12m
+kube-system                       coredns-565d847f94-92f4k                           1/1     Running   0          58m
+kube-system                       coredns-565d847f94-kgddm                           1/1     Running   0          58m
+kube-system                       etcd-florin-control-plane                          1/1     Running   0          58m
+kube-system                       kindnet-p8vkv                                      1/1     Running   0          58m
+kube-system                       kube-apiserver-florin-control-plane                1/1     Running   0          58m
+kube-system                       kube-controller-manager-florin-control-plane       1/1     Running   0          58m
+kube-system                       kube-proxy-jmxsg                                   1/1     Running   0          58m
+kube-system                       kube-scheduler-florin-control-plane                1/1     Running   0          58m
+local-path-storage                local-path-provisioner-684f458cdd-kw2xz            1/1     Running   0          58m
+
+``` 
+
+Now, let's onboard the `guilder` cluster:
+
+```shell
+kubectl kcp-edge prep-for-cluster --imw root:example-imw guilder env=prod
+```
+
+Apply the created edge syncer manifest:
+
+```shell
+kubectl --context kind-guilder apply -f guilder-syncer.yaml
+```
+
+
+### d. Create and deploy the Apache Server workload into florin and guilder clusters
+
+Create the `EdgePlacement` object for your workload. Its “where predicate” (the locationSelectors array) has one label selector that matches the Location objects (`florin` and `guilder`) created earlier, thus directing the workload to both edge clusters.
+
+In the `example-wmw` workspace create the following `EdgePlacement` object: 
+  
+```console
+kubectl ws root:my-org:example-wmw
+
+kubectl apply -f - <<EOF
+apiVersion: edge.kcp.io/v1alpha1
+kind: EdgePlacement
+metadata:
+  name: edge-placement-c
+spec:
+  locationSelectors:
+  - matchLabels: {"env":"prod"}
+  namespaceSelector:
+    matchLabels: {"common":"si"}
+  nonNamespacedObjects:
+  - apiGroup: apis.kcp.io
+    resources: [ "apibindings" ]
+    resourceNames: [ "bind-kube" ]
+  upsync:
+  - apiGroup: "group1.test"
+    resources: ["sprockets", "flanges"]
+    namespaces: ["orbital"]
+    names: ["george", "cosmo"]
+  - apiGroup: "group2.test"
+    resources: ["cogs"]
+    names: ["william"]
+EOF
+```
+
+Put the prescription of the HTTP server workload into the WMW. Note the namespace label matches the label in the namespaceSelector for the EdgePlacement (`edge-placement-c`) object created above. 
+
+
+```console
+kubectl apply -f - <<EOF
```

Comment:
I find copy-and-paste of large volumes of material somewhat painful.  We could _both_ (a) direct the user to issue a one-liner that uses a file in `examples/` AND (b) display the contents of that file right where a reader cares about them.

Reply:
We want to show the contents of the yaml manifest to the user. I find a contradiction  in the options a) and b) stated above. We should do either a) or b) but not both

Reply:
I like the copy-and-paste (I have seen also in other repos)